crates/edr_provider/src/test_utils.rs

```diff
@@ -38,12 +41,22 @@ pub fn create_test_config_with_impersonated_accounts(
         })
         .collect();
 
+    let mut fork = None;
+    if forked {
+        fork = Some(ForkConfig {
+            json_rpc_url: get_alchemy_url(),
+            // Random recent block for better cache consistency
+            block_number: Some(18_725_000),
+            http_headers: None,
+        });
+    }
```

Comment:
```suggestion
    let fork = if forked {
        Some(ForkConfig {
            json_rpc_url: get_alchemy_url(),
            // Random recent block for better cache consistency
            block_number: Some(18_725_000),
            http_headers: None,
        });
    } else {
        None
    };
```

Reply:
I prefer this pattern, as it avoids the variable from being mutable.

Reply:
Good call, thanks!